src/Hosting/SystemdServices/src/ServiceHelpers.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+
+namespace Microsoft.Extensions.Hosting.SystemdServices
+{
+    /// <summary>
+    /// Helper methods for systemd Services.
+    /// </summary>
+    public static class ServiceHelpers
```

Comment:
`SystemdHelpers`? As `ServiceHelpers` is quite generic.

Can this class be `internal`?

Reply:
I took the same scope and name as the class from the WindowsServices package.

Reply:
Yea it’s unfortunate we expose that method with such a generic name in the windows service package as well

Reply:
Can this be fixed or is it too late?

Reply:
It can be fixed but https://github.com/dotnet/corefx/issues/36842 isn't being fixed so we need something.

Reply:
Aside from that, I mean can the name be changed to `SystemdHelpers` and `WindowsServiceHelpers`?

Thinking loud: Should https://github.com/dotnet/corefx/issues/36842 be extended for `IsRunningSystemdService`?

Reply:
I think this should be `SystemdHelpers` and the Windows Service one should be `WindowsServicesHelpers` or something.

Reply:
Q: I've been naming things here 'SystemdService' because the Windows package names things 'WindowsService'. The Service suffix could be dropped, e.g. `UseSystemdService` vs `UseSystemd`. I don't have a strong preference for either.

Reply:
I'm fine with `UseSystemd`. It's `UseWindowsService` because "Windows Service" is the kind of executable, not necessarily because it's the "Windows" "background service" concept :). Plus `UseWindows` wouldn't be sufficient there.